client/src/components/footer/Footer.spec.js

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from '@testing-library/react'
+import Footer from './Footer'
+
+describe('Footer', () => {
+  it('renders', () => {
+    render(<Footer />)
+
+    const text = screen.getByText('FOOTER')
+    expect(text).toBeInTheDocument()
```

Comment:
You could also write `expect(screen.getByText('FOOTER').toBeInTheDocument()` (as far as I know).